docs/cloud-guidance/azure/ukho-azure-tagging.md

```diff
@@ -0,0 +1,28 @@
+# UKHO Azure Tagging Strategy v0.1 DRAFT
+
+## How to use this document
+
+This document outlines which tags should be applied when creating resources in the cloud. Use the table below to ensure you are conforming to the UKHO tagging strategy.
+
+## How to contribute to this document
+
+The owner of this document is Digital Operations. Please submit suggestions/changes/amendments vie a pull request and inform Digital Operations.
```

Comment:
Do we have to own it? Does it need an owner? Should it not be owned by someone else?(Azure Guild? ) If we do have to own, I suggest also adding a codeowners file with `/docs/cloud-guidance/azure/ukho-azure-tagging.md @UKHO/digital-ops ` within it.

Reply:
Yes we own it, it needs an owner and it's us. Though we'd expect much contribution from Azure guild

Reply:
OK :(